rust_src/src/fileio.rs

```diff
@@ -3,9 +3,22 @@ use std::path;
 
 use remacs_macros::lisp_fn;
 
-use lisp::defsubr;
+use lisp::{defsubr, LispObject};
+use math::{arithcompare, ArithComparison};
 use multibyte::LispStringRef;
 
+/// Return t if (car A) is numerically less than (car B).
+#[lisp_fn]
```

Comment:
Can't you specify each as a `LispCons` here?

Reply:
Ah, I didn't know that was possible. Done.

Reply:
Arguments and return values are handled by the lisp_fn macro.